data/korath/korath ships.txt

```diff
@@ -387,6 +387,169 @@ ship "'olofez"
+	turret 88.5 -150 "Warder Anti-Missile"
+	turret -110.5 -142 "Warder Anti-Missile"
+	turret 110.5 -142 "Banisher Grav-Turret"
+	turret 49.5 -53.5 "Langrage Hyper-Heaver"
+	turret -64 28 "Warder Anti-Missile"
+	turret 64 28 "Banisher Grav-Turret"
+	turret -64 50 "Banisher Grav-Turret"
+	turret 64 50 "Warder Anti-Missile"
+	bay "Fighter" -119.5 -179
+		angle 30
+	bay "Fighter" 119.5 -179
+		angle 330
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "Korath Refuges rescue the most vulnerable of Korath from battle. Onboard, there are incubators, nurseries, hospitals, and vast stores of medicine. When Korath are attacked, these ships are quick to respond."
+
+
+ship "Ik'atilajej"
+	sprite "ship/ik-atilajej"
+	thumbnail "thumbnail/ik-atilajej"
+	attributes
+		category "Utility"
+		"cost" 109375000
+		"shields" 72700
+		"hull" 93000
+		"required crew" 957
+		"bunks" 1933
+		"mass" 5380
+		"drag" 58
+		"heat dissipation" .29
+		"fuel capacity" 800
+		"cargo space" 845
+		"outfit space" 1829
+		"weapon capacity" 680
+		"engine capacity" 290
+		"spinal mount" 1
+		weapon
+			"blast radius" 600
+			"shield damage" 9000
+			"hull damage" 4800
+			"hit force" 20000
+	outfits
+		"Microbot Factory"
+		"Shunt-Strike Turret" 5
+		"Grab-Strike Turret" 4
+		"Warder Anti-Missile" 5
+
+		"Triple Plasma Core"
+		"Double Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
+		"Korath Repeater Rifle"  957
+		"Microbot Defense Station" 20
+
+		"Afterburner (Lunar Class)"
+		"Reverser (Lunar Class)"
+		"Steering (Stellar Class)"
+		"Thruster (Stellar Class)"
+		"Hyperdrive"
+
+	"reverse engine" -105 -190.5
+		zoom 0.8
+	"reverse engine" 105 -190.5
+		zoom 0.8
+	engine -84 231.5
+		zoom 0.8
+	engine 84 231.5
+		zoom 0.8
+	engine -61 262.5
+	engine 61 262.5
+	engine -27 249.5
+		zoom 0.8
+	engine 27 249.5
+		zoom 0.8
+	gun 0 -190 "Microbot Factory"
+	turret -90.5 -176 "Grab-Strike Turret"
+	turret 90.5 -176 "Grab-Strike Turret"
+	turret -95 -156 "Grab-Strike Turret"
+	turret 95 -156 "Grab-Strike Turret"
+	turret -51.5 -98.5 "Shunt-Strike Turret"
+	turret 51.5 -98.5 "Warder Anti-Missile"
+	turret 0 -94.5 "Shunt-Strike Turret"
+	turret 0 14 "Shunt-Strike Turret"
+	turret -145.5 83 "Shunt-Strike Turret"
+	turret 145.5 83 "Warder Anti-Missile"
+	turret -44.5 113 "Warder Anti-Missile"
+	turret 44.5 113 "Warder Anti-Missile"
+	turret -139.5 178 "Warder Anti-Missile"
+	turret 139.5 178 "Shunt-Strike Turret"
+	explode "tiny explosion" 180
+	explode "small explosion" 80
+	explode "medium explosion" 90
+	explode "large explosion" 60
+	explode "huge explosion" 25
+	"final explode" "final explosion large"
+	description "The Refabricator is a factory ship. Its microbot-run factory produces basic living supplies and ship parts, and vast systems cores in battle."
```

Comment:
```suggestion
	description "For most of their manufacturing needs, the Korath rely, as even some terrestrial species do, upon their space station and asteroid infrastructure. Because they do not have planets to evacuate to, however, the acceptability for failure and downtime is slim to nonexistent. Should equipment aboard a station or a world-ship fail, the Ikatila'ej, named for an inventor famously proficient in a variety of fields, is called in to provide critical support. Its microbot-run factory produces basic living supplies, machine parts, and systems cores, and may assist in continuing stalled production."
	description "	When not assisting with damaged ships and stations, or replenishing its own stores, the Ikatila'ej serves as an all-in-one mining, refining, and factory ship, rapidly turning asteroids into whatever products the civilian economic planning committees have determined to be in the most dire need."
```